src/docs/getting_started.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Getting Started
+---
+
+# Getting started
+
+In this section, we provide a high level overview on how to get started with Eclipse Theia and link to respective sections to read.
+Eclipse Theia is a platform for building custom Cloud & Desktop IDEs and tools with modern web technologies. Eclipse Theia is not a tool itself, but there are many tools which are built upon Theia. Theia as a project provides a template tool called [Eclipse Blueprint](https://theia-ide.org/docs/blueprint_download/) to try Theia and get started with building your own tool. Please have a look at the project goals for more details!
+
+## Frequently asked questions
+
+If you are unsure what Theia is, how it compares to other technologies or how to migrate Eclipse-based tools to it, please have a look at [this FAQ for Eclipse Theia](https://eclipsesource.com/blogs/2019/12/24/eclipse-theia-ide-faq/).
+
+## Try/Launch Eclipse Theia
+
+You cannot directly launch Theia as it is a platform. You can create and launch [products based on Theia](https://theia-ide.org/docs/composing_applications/). The project provides one example tool called [Eclipse Blueprint](https://theia-ide.org/docs/blueprint_download/) that you can directly download and use. Further, it is very easy to [create your own product based on Theia](https://theia-ide.org/docs/composing_applications/). Please also refer to [this article](https://eclipsesource.com/de/blogs/2019/09/25/how-to-launch-eclipse-theia/) highlighting the various options available to [launch/try Eclipse Theia and Theia-based products](https://eclipsesource.com/de/blogs/2019/09/25/how-to-launch-eclipse-theia/).
```

Comment:
I actually think we use platform more often, e.g. on the front page: "An Open, Flexible and Extensible Cloud & Desktop IDE Platform". It is also the more common term for me, e.g. Rich Client Platform, Eclipse Tools Platform, etc.

Reply:
The discussion we had was after `platform` was added to the website in order to clarify what to use (https://github.com/eclipse-theia/theia-website/pull/102) as I described in the issue but we had never updated:

> I have taken the liberty of updating the [website](https://theia-ide.org/) to reflect that we are a platform since it previously only described Theia as "Cloud & Desktop IDE".
>
> What do others think, should we use platform or framework in official documentation?

Not related necessarily to this change but I do think we should align in the future, and the consensus was to use framework as in the documentation present in the sources.